Formula/gping.rb

```diff
@@ -1,8 +1,9 @@
 class Gping < Formula
   desc "Ping, but with a graph"
   homepage "https://github.com/orf/gping"
-  url "https://github.com/orf/gping/archive/v1.2.0.tar.gz"
-  sha256 "2379d2d5c3e301140d9c65c4dcc2b99602acf511b2798f45009af4c1101a0716"
+  url "https://github.com/orf/gping/archive/v1.2.0-post.tar.gz"
```

Comment:
cc @orf not sure if the version scheme is intended. 😅

Reply:
Argh, I did a -post release just after the 1.2.0 release to include a MUSL binary.

Can we exclude this from livecheck? Or is there a convention I should use instead of "-post"?

Reply:
yeah, we can definitely exclude the `-post` tag for sure, just to confirm a bit :)

Reply:
update livecheck on this.
https://github.com/Homebrew/homebrew-core/pull/66454